src/Impl/Widget.h

```diff
@@ -0,0 +1,61 @@
+#ifndef WIDGET_H
+#define WIDGET_H
+
+#include "Plug/Widget.h"
+
+/**
+ * @brief Usable widget
+ *
+ */
+class Widget : public plug::Widget
+{
+public:
+  explicit Widget(const plug::LayoutBox& box);
+  Widget(const Widget& widget);
+  Widget& operator=(const Widget& widget);
+  ~Widget() override;
+
+  void draw(plug::TransformStack& stack, plug::RenderTarget& target) override {}
```

Comment:
Maybe render purple rectangle here? Or keep this class abstract and do not implement this method at all.

Reply:
Why have some absurd logic and have to erase it every time it is not needed?
If you want dedicated debug renderer then implement it as dedicated debug renderer.

Reply:
Do not provide useless default implementation then.